src/MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.TabControl.xaml

```diff
@@ -31,57 +31,83 @@
         <ControlTemplate TargetType="{x:Type TabControl}">
           <DockPanel KeyboardNavigation.TabNavigation="Local">
             <wpf:ColorZone x:Name="PART_HeaderZone"
-                           VerticalAlignment="Stretch"
-                           Panel.ZIndex="1"
-                           wpf:ElevationAssist.Elevation="{TemplateBinding wpf:ElevationAssist.Elevation}"
-                           Background="{TemplateBinding wpf:ColorZoneAssist.Background}"
-                           DockPanel.Dock="Top"
-                           Focusable="False">
-              <ScrollViewer wpf:ScrollViewerAssist.BubbleVerticalScroll="True"
-                            wpf:ScrollViewerAssist.SupportHorizontalScroll="True"
+                         VerticalAlignment="Stretch"
+                         Panel.ZIndex="1"
+                         wpf:ElevationAssist.Elevation="{TemplateBinding wpf:ElevationAssist.Elevation}"
+                         Background="{TemplateBinding wpf:ColorZoneAssist.Background}"
+                         DockPanel.Dock="Top"
+                         Focusable="False">
+              <Grid>
```

Comment:
Rather than this grid with both sets of controls. What if we apply a style trigger to the ColorZone and swap out its child content based on the AP?